app/controllers/admin/event_schedules_controller.rb

```diff
@@ -31,5 +31,11 @@ def destroy
     def event_schedule_params
       params.require(:event_schedule).permit(:schedule_id, :event_id, :room_id, :start_time)
     end
+
+    def parse_errors(event_schedule)
+      title = event_schedule.event.try(:title).present? ? event_schedule.event.title : 'The event'
```

Comment:
It should, but it is an error, if `.save`, `.update` or `.destroy` fails maybe a validation has failed, so I think we can not assume that event and title are present.